site/content/docs/main/sonobuoy-query.md

```diff
@@ -0,0 +1,18 @@
+# New `sonobuoy query` command
+
+## Intro
+
+- `sonobuoy query` command runs queries against our cluster and helps in debugging by providing useful information about the cluster which helps us to find bugs and fix them before they can cause any severe harm to the cluster . These queries were run from the aggregator pod (made through sonobuoy run) in the past. But now we can run them locally, even without running sonobuoy or bother launching pods and waiting for processing. We have to just run sonobuoy query.
```

Comment:
- Replace "queries against our cluster" with "queries against your cluster"
- You have an extra space before the period in `severe harm to the cluster .`
- nit: when referring to things that are code/commands use backticks to format them, so "sonobuoy run" should become `sonobuoy run`
- "But now we can run..." -> "You can run them..."
- "without running sonobuoy" should capitalize Sonobuoy
- Last line I would recommend trimming down: "But now we can run them locally, even without running sonobuoy or bother launching pods and waiting for processing. We have to just run sonobuoy query." -> "Now you can run them locally without running Sonobuoy or launching pods. Just run `sonobuoy query` which is much faster and convenient."